packages/shared/src/utils/platform/platform-context.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export const PlatformContext = React.createContext(null);
```

Comment:
this looks like only isolated to crypto, why not making platform store instead in core, instead of adding context in shared?

Reply:
We do use this in other packages, e.g. `Account` is now required to use this. That is why it has been added here.